Sources/ArgumentParser/Usage/UsageGenerator.swift

```diff
@@ -114,29 +114,11 @@ extension ArgumentDefinition {
   }
   
   var sortedNames: [Name] {
-    return names
```

Comment:
I missed that one. I assume, It will be the same for usage. Right?
If a single-dash long name comes first,
```
struct Example: ParsableCommand {
  @Option(name: [.short, .customLong("other", withSingleDash: true), .customLong("reminder"), .long])
  var example: Int
}
```

Then, usage will be like this
```
USAGE: example -other <other>
```

Reply:
Right! We should prefer the first long name declared, whether it has one or two dashes.